problem-matchers.json

```diff
@@ -0,0 +1,21 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "deno-lint",
+      "pattern": [
+        {
+          "regexp": "^(?:\\x1B\\[[0-9;]*[a-zA-Z])*(warning|warn|error)(?:\\[(\\S*)\\])?(?:\\x1B\\[[0-9;]*[a-zA-Z])*: (.*?)(?:\\x1B\\[[0-9;]*[a-zA-Z])*$",
```

Comment:
> (warning|warn|error)

I know that there is at least an `error` in the severity of `deno lint`, but I am not exactly sure what other severities exist. If anyone has exact information, please let me know.